app/services/users.js

```diff
@@ -0,0 +1,18 @@
+const user = require('../models').user,
+  logger = require('../logger'),
+  errors = require('../errors'),
+  bcryptService = require('../services/bcrypt');
+
+exports.create = userFields =>
+  user
+    .create({ ...userFields, password: bcryptService.encryptPassword(userFields.password) })
```

Comment:
This is wrong, the services only comunicate with controller. Move the logic from encrypt for controller.